integrations/spark/src/main/java/marquez/spark/agent/MarquezContext.java

```diff
@@ -33,9 +33,11 @@ public MarquezContext(ArgumentParser argument) throws URISyntaxException {
   public void emit(LineageEvent event) {
     try {
       // Todo: move to async client
+      log.debug("Posting LineageEvent {}", event);
       ResponseMessage resp = client.post(lineageURI, event);
       if (!resp.completedSuccessfully()) {
-        log.error("Could not emit lineage.", new MarquezHttpException(resp, resp.getError()));
+        log.error("Could not emit lineage: {}", OpenLineageClient.createMapper().writeValueAsString(event),
```

Comment:
We should just create one object mapper for all requests

Reply:
Fixed.